pkgs/tools/text/gawk/default.nix

```diff
@@ -57,9 +68,14 @@ stdenv.mkDerivation rec {
 
   inherit doCheck;
 
-  postInstall = ''
-    rm "$out"/bin/gawk-*
-    ln -s gawk.1 "''${!outputMan}"/share/man/man1/awk.1
+  postInstall = (if interactive then ''
+    remove-references-to -t "$NIX_CC" -- "$out"/bin/gawkbug
+    patchShebangs --host -- "$out"/bin/gawkbug
+  '' else ''
+    rm -- "$out"/bin/gawkbug
+  '') + ''
+    rm -- "$out"/bin/gawk-*
+    ln -s -- gawk.1 "''${!outputMan}"/share/man/man1/awk.1
```

Comment:
```suggestion
    installManPage gawk.1
```

Reply:
@SuperSandro2000, the manpage is already installed, this just creates a symlink/alias between awk and gawk. 😅

Reply:
Rebased and also updated the PR so that these lines no longer show as added/changed.